doc/source/whatsnew/v1.0.0.rst

```diff
@@ -3,38 +3,18 @@
 What's new in 1.0.0 (??)
 ------------------------
 
-.. warning::
-
-   Starting with the 1.x series of releases, pandas only supports Python 3.6.1 and higher.
+These are the changes in pandas 1.0.0. See :ref:`release` for a full changelog
+including other versions of pandas.
 
 New Deprecation Policy
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Starting with Pandas 1.0.0, pandas will adopt a version of `SemVer`_.
```

Comment:
Is there a reason you removed this? (just too long?)

I personally find it interesting context (it's also not that elaborate in the actual policy text)